learn/src/main/java/org/navgurukul/learn/courses/db/models/Exercise.kt

```diff
@@ -10,33 +10,33 @@ import java.io.Serializable
 @Entity(tableName = "course_exercise", primaryKeys = ["id", "lang"])
 @JsonClass(generateAdapter = true)
 data class Exercise(
-    @Json(name = "content")
-    var content: MutableList<ExerciseSlugDetail>? = mutableListOf(),
-    @Json(name = "course_id")
+        @Json(name = "content")
+    var content: MutableList<BaseCourseContent>? = mutableListOf(),
+        @Json(name = "course_id")
     var courseId: String? = "",
-    @Json(name = "github_link")
+        @Json(name = "github_link")
     var githubLink: String? = "",
-    @Json(name = "id")
+        @Json(name = "id")
     @ColumnInfo(name = "id")
     var id: String = "",
-    @Json(name = "name")
+        @Json(name = "name")
     var name: String? = "",
-    @Json(name = "parent_exercise_id")
+        @Json(name = "parent_exercise_id")
     var parentExerciseId: String? = "",
-    @Json(name = "review_type")
+        @Json(name = "review_type")
     var reviewType: String? = "",
-    @Json(name = "sequence_num")
+        @Json(name = "sequence_num")
     var sequenceNum: String? = String(),
-    @Json(name = "slug")
+        @Json(name = "slug")
     var slug: String? = "",
-    @Json(name = "solution")
+        @Json(name = "solution")
     var solution: String? = String(),
-    @Json(name = "submission_type")
+        @Json(name = "submission_type")
     var submissionType: String? = String(),
-    var lang: String = "en",
-    @Ignore
+        var lang: String = "en",
+        @Ignore
     var number: Int? = 0,
 
-    @Json(name = "courseName")
+        @Json(name = "courseName")
```

Comment:
Lets remove defaults from here, they should be null if the server doesn't send these keys